kmm/kmm-post-data/src/commonMain/kotlin/com/gchristov/newsfeed/kmmpostdata/PostRepository.kt

```diff
@@ -6,14 +6,14 @@ interface PostRepository {
     suspend fun post(
         postId: String,
         postMetadataFields: String = "main,body,headline,thumbnail"
-    ): DecoratedPost
+    ): Post
 
-    suspend fun redecoratePost(post: DecoratedPost): DecoratedPost
-
-    suspend fun cachedPost(postId: String): DecoratedPost?
+    suspend fun cachedPost(postId: String): Post?
 
     suspend fun clearCache(postId: String)
 
+    suspend fun cachePost(decoratedPost: DecoratedPost)
```

Comment:
Lets change this to cache a `Post` rather than a `DecoratedPost` and we'll update in a separate future PR, as this is a separate piece of work.

Reply:
@gchristov  I attempted the change and seems a bit too invasive at this point. Given the size of the PR I would suggest bundling it with the DecoratedPost cache (issue to be created?)

Reply:
Yep that makes sense, lets keep and change in a separate PR. I think I'll make issues for these things which will be extracted from the rough plan doc in Slack.